fuuk/people/models/grant.py

```diff
@@ -22,12 +25,16 @@ def __unicode__(self):
 
 
 class Grant(models.Model):
-    author = models.ForeignKey(Person, verbose_name=_('Applicant'))
+    editor = models.ForeignKey(User, blank=True, null=True, on_delete=models.CASCADE)
+    investigator_prefix = models.CharField(max_length=20, blank=True, null=True)
+    investigator_first_name = models.CharField(max_length=50)
+    investigator_last_name = models.CharField(max_length=50)
+    investigator_suffix = models.CharField(max_length=20, blank=True, null=True)
+    investigator_institution = models.ForeignKey(Institution, blank=True, null=True, on_delete=models.CASCADE)
```

Comment:
Starting with django 1.9 it is actually recommended to have `on_delete` as a second positional argument and it will be required with django 2.0...
https://docs.djangoproject.com/en/1.9/ref/models/fields/#foreignkey